mmseg/models/backbones/vit.py

```diff
@@ -225,6 +225,7 @@ def __init__(self,
                  norm_eval=False,
                  with_cp=False,
                  pretrain_style='timm',
+                 with_image_shape=False,
```

Comment:
`with_image_shape` -> `with_spatial_size`?
The spatial size of the corresponding feature map.

Reply:
Add docstring for this `arg`